kumascript/macros/geckoRelease.ejs

```diff
@@ -80,7 +80,7 @@ switch(arg)
     break;
 }
 if (geckoRelease != "") {
-  geckoRelease = string.Join(geckoRelease, plus + ' / ') + plus;
+  geckoRelease = geckoRelease.join(plus + ' / ') + plus;
```

Comment:
It's not your fault but this is horrible :)
You look at the if statement above and based on that you think `geckoRelease` must be a string because the if statement checks if it's not a particular string. But strings don't have a `.join()`.

Reply:
I don't like it either, so I refactored it a bit. Does it look better now?